src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({
+  post,
+  comments = post.comments,
+  user = post.user,
+}) => {
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
```

Comment:
user can be undefined, so it's better to add the check
```suggestion
          {user && <UserInfo user={user} />}
```